src/shared/lib/markDownRules.tsx

```diff
@@ -233,6 +233,15 @@ const markDownRules: RenderRules = {
       </Box>
     );
   },
+  table: (node, children) => {
+    const { width: screenWidth } = Dimensions.get('screen');
+
+    return (
+      <Box key={node.key} width={screenWidth - 32}>
```

Comment:
It is not clear what is 32. Please create a variable that describes it. 
For example, const PADDING_X = 32